editor/src/clj/editor/analytics.clj

```diff
@@ -63,46 +62,27 @@
   (and (string? value)
        (.matches (re-matcher cid-regex value))))
 
-(defn- valid-uid? [value]
```

Comment:
There is no user id anymore (and surely hasn't been for some time since we no longer authenticate the user)